django_ibmi/base.py

```diff
@@ -287,27 +361,25 @@ def _cursor( self, settings = None ):
                     self.settings = settings
                     
                 self.connection = self.get_new_connection(self.get_connection_params())
-                cursor = self.databaseWrapper._cursor(self.connection)
+                cursor = DB2CursorWrapper(self.connection)
                 
                 if( djangoVersion[0:3] <= ( 1, 2, 2 ) ):
                     connection_created.send( sender = self.__class__ )
                 else:
                     connection_created.send( sender = self.__class__, connection = self )
             else:
-                cursor = self.databaseWrapper._cursor( self.connection )  
+                cursor = DB2CursorWrapper(self.connection)
             return cursor
     else:
         def create_cursor( self , name = None):
-            return self.databaseWrapper._cursor( self.connection )
+            return DB2CursorWrapper(self.connection)
             
         def init_connection_state( self ):
             pass
-        
+
         def is_usable(self):
-            if self.databaseWrapper.is_active( self.connection ):
-                return True
-            else:
-                return False
+            # Implemented incorrectly
```

Comment:
I think this would need to return `True` for now, no? Without returning anything, `None` is returned, which is falsy in a boolean context.